### PR TITLE
Adding forward translator includes for ground temperature objects

### LINKED ISSUE #2237: Site:GroundTemperature:Deep not forward translating
In the newly added `SiteGroundTemperatureDeep` object, it's not getting forward translated into the idf. It appears that I forgot to put the appropriate include [here](https://github.com/NREL/OpenStudio/blob/792d26fcdd32e3b412079178a29983e640960eb0/openstudiocore/src/sdd/ForwardTranslator.cpp#L125).

Is there some sort of unit testing for the forward translator that I should be doing? @macumber

### PR DESCRIPTION
**Pull request by nmerket**
Fixes #2237

### CODE REVIEW

**Comment by nmerket** [comment]
Just tested this and it's working now. Ready to merge.

File: openstudiocore/src/energyplus/ForwardTranslator.cpp [hunk 1]
```diff
@@ -2122,7 +2122,7 @@ boost::optional<IdfObject> ForwardTranslator::translateAndMapModelObject(ModelOb
       model::ShadingControl shadingControl = modelObject.cast<ShadingControl>();
       retVal = translateShadingControl(shadingControl);
       break;
-    }
```
Comment (macumber) [inline openstudiocore/src/energyplus/ForwardTranslator.cpp]: @nmerket in the future you might set your editor to ignore trailing whitespace, just makes the diffs harder to compare

Comment (shorowit) [inline openstudiocore/src/energyplus/ForwardTranslator.cpp]: @macumber did you know this? https://github.com/blog/967-github-secrets

Comment (macumber) [inline openstudiocore/src/energyplus/ForwardTranslator.cpp]: @shorowit no i did not but that is awesome!

https://github.com/NREL/OpenStudio/pull/2254/files?w=1

**Comment by macumber** [comment]
Thanks @nmerket !

**Pull request merged**